tests/integration/plugins/test_dotnet_v2.py

```diff
@@ -0,0 +1,136 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import subprocess
+import textwrap
+from pathlib import Path
+
+import yaml
+from craft_parts import LifecycleManager, Step
+from craft_parts.plugins import dotnet_v2_plugin
+from overrides import override
+
+
+def test_dotnet_plugin(new_dir, partitions):
+    # pylint: disable=line-too-long
+    parts_yaml = textwrap.dedent(
+        """
+        parts:
+          foo:
+            source: .
+            plugin: dotnet
+            dotnet-self-contained: true
+            build-environment:
+              - PATH: $CRAFT_STAGE/sdk:$PATH
+            after: [dotnet-deps]
+          dotnet-deps:
+            plugin: dump
+            source: https://download.visualstudio.microsoft.com/download/pr/d2abdb4c-a96e-4123-9351-e4dd2ea20905/e8010ae2688786ffc1ebca4ebb52f41b/dotnet-sdk-8.0.406-linux-x64.tar.gz
+            source-checksum: sha512/d6fdcfebd0df46959f7857cfb3beac7de6c8843515ece28b24802765fd9cfb6c7e9701b320134cb4907322937ab89cae914ddc21bf48b9b6313e9a9af5c1f24a
+            override-build: |
+              # TODO: find out why this is a problem with "organize".XS
```

Comment:
What's the problem?

Reply:
I'm afraid this was a left-over from the original .NET plugin integration tests. I copied over the same [parts definition](https://github.com/canonical/craft-parts/blob/main/tests/integration/plugins/test_dotnet.py#L44) from there to create the tests for the new plugin.

Reply:
I see.  Since this isn't for the dotnet plugin itself, I'm fine with that for now. 

One day it would be nice to try using `organize` and see if it's still failing.